src/tests/ftest/datamover/copy_procs.py

```diff
@@ -69,7 +69,7 @@ def test_copy_procs(self):
             Create pool.
             Crate POSIX container1 and container2 in pool.
             Create a single 100M file in container1 using ior.
-        :avocado: tags=all,pr,hw,small,datamover
+        :avocado: tags=all,datamover,daily_regression,hw
```

Comment:
@brianjmurrell Was it intentional for `small` to be removed from the tags?

Reply:
Most likely just a merge conflict mis-resolved given that the move to small was only very shortly prior to this landing.  Will resolve in a PR I am about to push to fix something else.

Reply:
@brianjmurrell Sure, no worries. If you fix it, that would be great. Otherwise, I'll update it myself with my next PR in the area. Thanks!